### ISSUE TITLE
Inflector rules in initializers no longer apply

### ISSUE THREAD

**Issue by jrjohnson** [emberjs/data#3730]
This has been working but is now failing with ember-data 1.13.11,.  I'm not sure exactly when it started to fail, I'm going to try building a bunch of versions to see if I can pin it down.  Unless this is known, or I'm doing it wrong, then I won't spend the time.

app/initializers/inflector.js
```javascript
import Ember from 'ember';

export function initialize(/* container, application */) {
  Ember.Inflector.inflector.uncountable('aamc-pcrs');
}

export default {
  name: 'inflector',
  initialize: initialize
};
```
I get `Error: No model was found for 'aamc-pcr'`

However it works fine if I move it into 
app/app.js
```javascript
Ember.Inflector.inflector.uncountable('aamc-pcrs');
```

The user of an initializer for this seems to be the generally accepted advice, but there is no clear recommendation in the guides so maybe I've been doing it wrong?  Or maybe something in the initializer chain is different.

**Comment by fivetanley** [comment]
Hey @jrjohnson, sorry for the late reply. I think this should be fixed as of 1.13.16. Please let me know if it does not. For more info, here's the PR: https://github.com/emberjs/data/pull/4143

**Comment by jrjohnson** [comment]
We are on 2.3.x now and were able to move this back into an inflector so I can't confirm.  I think this can be safely closed though.
